instrumentation/mongo/mongo-common/src/main/java/io/opentelemetry/javaagent/instrumentation/mongo/MongoClientTracer.java

```diff
@@ -130,10 +133,24 @@ private static BsonValue scrub(BsonValue origin) {
     return scrubbed;
   }
 
+  private static final Set<String> COMMANDS_WITH_COLLECTION_NAME_AS_VALUE =
+      new HashSet<>(asList(
+          "aggregate", "count", "distinct", "mapReduce", "geoSearch", "delete", "find", "killCursors",
```

Comment:
While a n allow-list is in some ways not future proof, in practice I think it's safe as the trend in MongoDB has been to use new initiating stages in aggregation pipelines to add new functionality (e.g. the `$changeStream` stage.